src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int OPERATION_INDEX = 0;
+    public static final int COUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
this method should have logic divided like this
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
```